packages/frontend/app/components/user-profile-bio.js

```diff
@@ -44,6 +45,40 @@ export default class UserProfileBioComponent extends Component {
 
   userSearchTypeConfig = new TrackedAsyncData(this.iliosConfig.getUserSearchType());
 
+  constructor() {
+    super(...arguments);
+
+    const user = this.args.user;
+
+    this.firstName = user.firstName;
+    this.middleName = user.middleName;
+    this.lastName = user.lastName;
+    this.campusId = user.campusId;
+    this.otherId = user.otherId;
+    this.email = user.email;
+    this.displayName = user.displayName;
+    this.pronouns = user.pronouns;
+    this.preferredEmail = user.preferredEmail;
+    this.phone = user.phone;
+  }
+
+  getAuthUser = modifier(() => {
+    if (this.userAuthentication) {
```

Comment:
i don't understand how/why this would work, but it does.  seems like modifiers react to component  property changes, like getters.

any chance this could be made more explicit? 

perhaps `userAuthentication` could be made input to the modifier. 
something like 

`getAuthUser = modifier((userAuthentication))` and  `{{this.getAuthUser this.userAuthentication}}`.